contracts/interfaces/modules/IERC1271Unsafe.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.27;
+
+interface IERC1271Unsafe {
```

Comment:
Can we rename this? "Unsafe" gives a bad signal. Maybe something like IERC1271Legacy or similar would be better

Reply:
I like legacy, yes